pom.xml

```diff
@@ -15,7 +15,7 @@
   </licenses>
 
   <artifactId>github-api</artifactId>
-  <version>1.72.2-SNAPSHOT</version>
+  <version>1.75-SNAPSHOT</version>
```

Comment:
Why are you jumping from 1.72.2 to 1.75? Does it always match the `github-api` version?